app/src/main/java/umc/OnAirMate/ui/pop_up/CreateRoomPopup.kt

```
@@ -0,0 +1,143 @@
+                editHandler.postDelayed(editRunnable!!, 300) // 300ms 디바운스
+            }
+        })
+        binding.etInputRoomTitle
+    }
+
+    // 팝업 띄우는 화면용 함수 -> 복붙하여 사용
+    private fun test(data : VideoData){
+        val dialog = CreateRoomPopup(data, object : CreateRoomCallback {
+            override fun onCreateRoom(body: CreateRoomRequest) {
+
+            }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)